frontend/src/components/TestCase.tsx

```
@@ -0,0 +1,48 @@
+                Input: <code>{input}</code>
+            </p>
+            <div className={styles.userInput}>
+                <p>Output:</p>
+                <input
+                    type="text"
+                    value={verified ? output : userOutput}
+                    onChange={handleChange}
+                    placeholder="Enter your output"
+                    disabled={verified}
+                />
+                <button onClick={handleCheck} disabled={verified}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)